src/lib.rs

```diff
@@ -250,4 +255,13 @@ mod tests {
             .unwrap(),
         );
     }
+
+    #[test]
+    fn convert() {
+        for mac in MacAddressIterator::new().unwrap() {
+            let name = name_by_mac_address(&mac).unwrap().unwrap();
+            let mac2 = mac_address_by_name(&name).unwrap().unwrap();
+            assert_eq!(mac, mac2);
+        }
+    }
```

Comment:
This should be moved down with the other tests at the bottom

Reply:
Where this should be moved to?

Reply:
At the end of the file there is a `#[cfg(test]` module, put it somewhere in there with the rest of the tests :)

Reply:
I think the test `convert` is added at the bottom of the `tests` module already.

Reply:
oh, duh! You're right, I wasn't paying attention to the line numbers and the Github diff made it look like it was inside of the `impl MacAddress` :facepalm: disregard this then, my bad!